src/js/internal/util/inspect.js

```diff
@@ -453,11 +436,14 @@ try {
 }
 
 const extractedSplitNewLines = value => {
-  if (typeof value === "string") {
-    return extractedSplitNewLinesFastPathStringsOnly(value) || extractedSplitNewLinesSlow(value);
+  const lines = typeof value === "string" ? StringPrototypeSplit(value, "\n") : RegExpPrototypeSymbolSplit(/\n/, value);
+  const last = ArrayPrototypePop(lines);
```

Comment:
Do we need to pop the last element of the array and then push it again? or can we check `lines[lines.length - 1] === '\n'`